ui/common/src/main/java/de/danoeh/antennapod/ui/common/ClipboardUtils.java

```diff
@@ -0,0 +1,72 @@
+package de.danoeh.antennapod.ui.common;
+
+import android.content.ClipData;
+import android.content.ClipboardManager;
+import android.content.Context;
+import android.os.Build;
+import android.view.View;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.StringRes;
+
+import com.google.android.material.snackbar.Snackbar;
+
+/**
+ * Utilities for commonly used clipboard functionality.
+ */
+
+public abstract class ClipboardUtils {
```

Comment:
Please remove the empty lines between javadoc and definition (everywhere)

```suggestion
 */
public abstract class ClipboardUtils {
```

Reply:
Done